common/changes/@bentley/presentation-components/presentation-typeName-lowercase_2021-03-02-13-38.json

```
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@bentley/presentation-components",
+      "comment": "Updated `createPropertyDescription` in `ContentBuilder` to set property's typename to lowercase.",
```

`ContentBuilder` is internal, so we shouldn't mention it here.
```suggestion
      "comment": "Always set [PropertyDescription.typename]($ui-abstract) to lowercase when creating content.",
```